Resources/views/full/file.html.twig

```diff
@@ -0,0 +1,34 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    {% set fileSize = content.getField( 'file' ).value.fileSize|ez_file_size( 4 ) %}
+    {% set fileName = content.getField( 'file' ).value.fileName %}
+    {% set downloadPath = path( ez_route( 'ez_content_download', { 'content': content, 'fieldIdentifier': 'file' } ) ) %}
+
+    <section class="content-view-full">
+        <article class="class-file">
+
+            <div class="attribute-header">
+                <h1>{{ ez_content_name( content ) }}</h1>
+            </div>
+
+            {% if not ez_is_field_empty( content, 'description' ) %}
+                <div class="attribute-long">
+                    {{ ez_render_field( content, 'description' ) }}
+                </div>
+            {% endif %}
+
+            <div class="attribute-file">
+                <p><a href="{{ downloadPath }}" class="btn btn-warning">{{ fileName }} {{ fileSize }}</a></p>
+            </div>
+
+            <div class="attribute-tags">
+                {{ ez_render_field( content, 'tags', { 'attr': { 'class' : 'tags-wrapper' } } ) }}
+            </div>
+
+            <div class="attribute-star-rating">
+                {{ render( controller( 'eZDemoBundle:Demo:viewStarRating', { 'location': location } ) ) }}
```

Comment:
yes, but what it does? i can't see a viewStarRating action at https://github.com/ezsystems/DemoBundle/blob/master/Controller/DemoController.php